### ISSUE TITLE
Make compatble with OC 2.x

### ISSUE THREAD

**Issue by kevinb456** [InaneCoding/Piwik-OpenCart-Ecommerce-VQMod#25]
Make compatible with latest version of OC 2.x (backwards compatible if possible).

**Comment by kevinb456** [comment]
Received email from Jozef Popiel with a contribution and reportedly working mod "Hi, First of all thanks for your great module. I needed a piwik module for Opencart 2.x and I saw that there are no modules for this version on OC extensions site. So I modified your module to make it compatible with OC 2.x. Basically it wasn't a big deal. I needed to slightly change admin/controller/module/piwik.php and rework admin/view/template/module/piwik.tpl to match new OC admin styles and templates pattern. I have also changed vqmod file. In OC 2 it is installed directly from the shop admin. Also the syntax is a little bit different (I used vqmod for OC 2, not the standard ocmod). ..."

Sent link of mod contents - need to investigate & test, then possibly release a new version using this.

**Comment by pinkslink** [comment]
any chance to get details to make compatible for OC2 ?
i be happy to test and report any issues if any found

**Comment by Oseexperts** [comment]
You can test my OC 2.x adaptation of this mod here:
https://github.com/Oseexperts/Piwik-OpenCart-Ecommerce-VQMod

You MUST have vqmod for OC 2.x installed to use this module. Do not use the built in Ocmod extension uploader in the store admin. Just upload the files like you would do for OC 1.5.x

**Comment by pinkslink** [comment]
thanks a lot, i will test on test site and then live site

**Comment by Oseexperts** [comment]
I use it on live site for few weeks already and havent spotted any major issues so far.

**Comment by kewh** [comment]
I have OC 2 installed and would like to use Piwik. Is there any progress on this?

**Comment by Oseexperts** [comment]
You can use the link provided by me 2 posts earlier.
It is stable, I have tested it on OC 2 so it should work OK for you. But in case of problems just write here or open an issue on my fork and I will gladly assist:
https://github.com/Oseexperts/Piwik-OpenCart-Ecommerce-VQMod/issues

**Comment by kewh** [comment]
Thanks, I’ll download this and give it a go. Thanks for your offer of help, I’ll let you know how I progress.

Keith

**Comment by kevinb456** [comment]
I have committed oseexperts changes to the develop branch, for integration into the main branch once tested. I know this has taken ages for me to look at (thanks for emails Joseph).

This eve I made further modifications (see 9be8300896cf4c10e9d2cbd95831d759e857372e) to fit in with the new OC2 formatting on the admin page. Gotta do a proper job :)

![image](https://cloud.githubusercontent.com/assets/4441138/7426485/52770cfe-efbe-11e4-9bda-3ca12e809a5d.png)

I am aiming to have this finished this weekend!
